requirements-dev.txt

```diff
@@ -4,3 +4,4 @@ pytest-cov
 pytest-xdist
 responses
 flake8
+hypothesis[django]
```

Comment:
That's setuptools "extras" syntax. We're using it here to say "install `hypothesis`, its dependencies, and any optional dependencies it declares under the name `django`".

You can define some optional dependencies in your `setup.py` (The docs at https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies show how) and then you use `package[extra]` instead of `package` as the dependency name to pull in the extra packages.